libcudacxx/codegen/generators/compare_and_swap.h

```diff
@@ -83,7 +83,6 @@ static inline _CCCL_DEVICE bool __cuda_atomic_compare_exchange(
   };
 
   constexpr size_t supported_sizes[] = {
-    16,
```

Comment:
Why are we dropping support here?

Was it effectively not supported or is that now covered by the other code paths

Reply:
Because 16b is emulated, we can just implement it ourselves and emulate over the full 32b intrinsic. Saves us from effectively using two CAS loops.